content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,280 @@
+    namespace/kube-system labeled
+    namespace/local-path-storage labeled
+    
+    
+    $ kubectl label --dry-run=server --overwrite ns --all \
+    pod-security.kubernetes.io/enforce=baseline
+    namespace/default labeled
+    namespace/kube-node-lease labeled
+    namespace/kube-public labeled
+    Warning: existing pods in namespace "kube-system" violate the new PodSecurity enforce level "baseline:latest"
+    Warning: etcd-psa-wo-cluster-pss-control-plane (and 3 other pods): host namespaces, hostPath volumes
+    Warning: kindnet-vzj42: non-default capabilities, host namespaces, hostPath volumes
+    Warning: kube-proxy-m6hwf: host namespaces, hostPath volumes, privileged
+    namespace/kube-system labeled
+    namespace/local-path-storage labeled
+    
+    
+    $ kubectl label --dry-run=server --overwrite ns --all \
+    pod-security.kubernetes.io/enforce=restricted
+    namespace/default labeled
+    namespace/kube-node-lease labeled
+    namespace/kube-public labeled
+    Warning: existing pods in namespace "kube-system" violate the new PodSecurity enforce level "restricted:latest"
+    Warning: coredns-7bb9c7b568-hsptc (and 1 other pod): unrestricted capabilities, runAsNonRoot != true, seccompProfile
+    Warning: etcd-psa-wo-cluster-pss-control-plane (and 3 other pods): host namespaces, hostPath volumes, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true
+    Warning: kindnet-vzj42: non-default capabilities, host namespaces, hostPath volumes, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true, seccompProfile
+    Warning: kube-proxy-m6hwf: host namespaces, hostPath volumes, privileged, allowPrivilegeEscalation != false, unrestricted capabilities, restricted volume types, runAsNonRoot != true, seccompProfile
+    namespace/kube-system labeled
+    Warning: existing pods in namespace "local-path-storage" violate the new PodSecurity enforce level "restricted:latest"
+    Warning: local-path-provisioner-d6d9f7ffc-lw9lh: allowPrivilegeEscalation != false, unrestricted capabilities, runAsNonRoot != true, seccompProfile
+    namespace/local-path-storage labeled
+    ```
+
+It seems like `privileged` Pod Security Standard when applied, shows no warnings
+for any namespaces. However, `baseline` and `restricted` standards both have
+more warnings specifically on `kube-system` namespace.
+
+# Setting modes, versions and standards
+
+For the purposes of this tutorial, `baseline` standard will be `enforced` as per
+`latest` version whereas `restricted` standard will be applied in
+`warn` and `audit` mode. Baseline Pod Security Standard provides a convenient
+middle ground that allows keeping the exemption list short and prevents known
+privilege escalations.
+
+However, to prevent pods from failing in `kube-system`, this namespace will be
+exempt from the cluster level Pod Security Standards applied.
```

Comment:
```suggestion
Additionally, to prevent pods from failing in `kube-system`, you'll exempt the namespace
from having Pod Security Standards applied.
```